### ISSUE TITLE
carbohydrate metabolic disorder vs carbohydrate metabolism disease

### ISSUE THREAD

**Issue by cmungall** [DiseaseOntology/HumanDiseaseOntology#371]
compare and contrast:

```
[Term]
id: DOID:2978
name: carbohydrate metabolic disorder
namespace: disease_ontology
alt_id: DOID:9434
def: "An inherited metabolic disorder that affect the catabolism and anabolism of carbohydrates." [url:http://en.wikipedia.org/wiki/Inborn_errors_of_carbohydrate_metabolism]
synonym: "disorder of carbohydrate transport and metabolism" EXACT []
synonym: "inborn carbohydrate metabolism disorder" EXACT [CSP2005:0551-8201]
synonym: "inborn errors of carbohydrate metabolism" EXACT []
xref: ICD9CM:271.8
xref: MESH:D002239
xref: UMLS_CUI:C0007001
xref: UMLS_CUI:C0029777
is_a: DOID:655  ! inherited metabolic disorder

[Term]
id: DOID:0050013
name: carbohydrate metabolism disease
namespace: disease_ontology
def: "An acquired metabolic disease that is characterized by abnormal carbohydrate metabolism." [url:http://www.nlm.nih.gov/medlineplus/metabolicdisorders.html]
subset: DO_FlyBase_slim  ! DO_FlyBase_slim
subset: DO_MGI_slim  ! DO_MGI_slim
is_a: DOID:0060158  ! acquired metabolic disease
```

it's not obvious from the labels, but these are both groupings for carbohydrate metabolism differentiated by acquired vs inherited. Many people do not read the definitions so it is important the labels are clear.

Note currently it's the acquired for that is in the DO and MGI subsets. Is this intentional? cc @sbello @FlyBase

I recommend:

 - changing the label
 - adding a grouping class for etiology-neutral carbohydrate metabolism disease
    - add GO OWL def
 - making explicit OWL differentiae for the two children
 - evaluating subclasses

This is the current grouping, at 'disease of metabolism':

![image](https://user-images.githubusercontent.com/50745/29252220-b7b8a820-8017-11e7-9704-8f9d3f0f2ede.png)

This is the descendants of the acquired form; note it includes OMIMs. Is this correct?

```
   is_a DOID:0050013 ! carbohydrate metabolism disease *** 
     is_a DOID:4194 ! glucose metabolism disease
      is_a DOID:0050884 ! triosephosphate isomerase deficiency
      is_a DOID:2018 ! hyperinsulinism
      is_a DOID:4195 ! hyperglycemia
       is_a DOID:10603 ! glucose intolerance
      is_a DOID:9351 ! diabetes mellitus
       is_a DOID:11714 ! gestational diabetes
       is_a DOID:11716 ! prediabetes syndrome
       is_a DOID:11717 ! neonatal diabetes mellitus
        is_a DOID:0060334 ! transient neonatal diabetes mellitus
        is_a DOID:0060638 ! neonatal diabetes mellitus with congenital hypothyroidism
        is_a DOID:0060639 ! permanent neonatal diabetes mellitus
       is_a DOID:9352 ! type 2 diabetes mellitus
        is_a DOID:10182 ! diabetic peripheral angiopathy
        is_a DOID:11712 ! lipoatrophic diabetes
        is_a DOID:1837 ! diabetic ketoacidosis
       is_a DOID:9744 ! type 1 diabetes mellitus
        is_a DOID:0110741 ! type 1 diabetes mellitus 2
        is_a DOID:0110742 ! type 1 diabetes mellitus 3
        is_a DOID:0110743 ! type 1 diabetes mellitus 4
        is_a DOID:0110744 ! type 1 diabetes mellitus 5
        is_a DOID:0110745 ! type 1 diabetes mellitus 6
        is_a DOID:0110746 ! type 1 diabetes mellitus 7
        is_a DOID:0110747 ! type 1 diabetes mellitus 8
        is_a DOID:0110749 ! type 1 diabetes mellitus 10
        is_a DOID:0110750 ! type 1 diabetes mellitus 11
        is_a DOID:0110751 ! type 1 diabetes mellitus 12
        is_a DOID:0110752 ! type 1 diabetes mellitus 13
        is_a DOID:0110753 ! type 1 diabetes mellitus 15
        is_a DOID:0110754 ! type 1 diabetes mellitus 17
        is_a DOID:0110755 ! type 1 diabetes mellitus 18
        is_a DOID:0110756 ! type 1 diabetes mellitus 19
        is_a DOID:0110757 ! type 1 diabetes mellitus 20
        is_a DOID:0110758 ! type 1 diabetes mellitus 21
        is_a DOID:0110759 ! type 1 diabetes mellitus 22
        is_a DOID:0110760 ! type 1 diabetes mellitus 23
        is_a DOID:0110761 ! type 1 diabetes mellitus 24
      is_a DOID:9993 ! hypoglycemia
```

Mouse models: http://www.informatics.jax.org/disease/DOID:0050013

and descendants of the inherited form:

```
    is_a DOID:2978 ! carbohydrate metabolic disorder *** 
     is_a DOID:0050728 ! glycogen metabolism disorder
      is_a DOID:2747 ! glycogen storage disease
       is_a DOID:0050579 ! glycogen storage disease XV
       is_a DOID:0050594 ! glycogen storage disease IX
        is_a DOID:0111040 ! glycogen storage disease IXd
        is_a DOID:0111041 ! glycogen storage disease IXb
        is_a DOID:0111042 ! glycogen storage disease IXa
        is_a DOID:0111043 ! glycogen storage disease IXc
       is_a DOID:0090101 ! lethal congenital glycogen storage disease of heart
       is_a DOID:11721 ! glycogen storage disease VII
       is_a DOID:2746 ! glycogen storage disease V
       is_a DOID:2748 ! glycogen storage disease III
       is_a DOID:2749 ! glycogen storage disease I
       is_a DOID:2750 ! glycogen storage disease IV
       is_a DOID:2751 ! glycogen storage disease VIII
       is_a DOID:2752 ! glycogen storage disease II
       is_a DOID:2754 ! glycogen storage disease VI
     is_a DOID:0060728 ! NGLY1-deficiency
     is_a DOID:10604 ! lactose intolerance
     is_a DOID:13317 ! hyperinsulinemic hypoglycemia
     is_a DOID:2862 ! glucosephosphate dehydrogenase deficiency
      is_a DOID:13628 ! favism
     is_a DOID:2977 ! primary hyperoxaluria
     is_a DOID:3649 ! pyruvate decarboxylase deficiency
     is_a DOID:3651 ! pyruvate carboxylase deficiency disease
     is_a DOID:5204 ! fructose-1,6-bisphosphatase deficiency
     is_a DOID:5212 ! congenital disorder of glycosylation
      is_a DOID:0050570 ! congenital disorder of glycosylation type I
      is_a DOID:0050571 ! congenital disorder of glycosylation type II
     is_a DOID:857 ! multiple carboxylase deficiency
      is_a DOID:856 ! biotinidase deficiency
      is_a DOID:859 ! holocarboxylase synthetase deficiency
     is_a DOID:9868 ! intestinal disaccharidase deficiency
     is_a DOID:9869 ! hereditary fructose intolerance syndrome
     is_a DOID:9870 ! galactosemia
      is_a DOID:14695 ! galactokinase deficiency
```

mouse models:
http://www.informatics.jax.org/disease/DOID:2978

**Comment by lschriml** [comment]
Hello Chris,
Thank you for your interest in improving the DO. 
I remember discussing this area of the DO a few years ago at EBI. 

As you have seen, disease nomenclature can be rather complicated. 
Keeping in mind of course, that each research community has defined, and refined,
  over time their nomenclature. 

Background: in disease nomenclature, disorder = inherited. 
  In the metabolism disease community, these labels are the two categories they define.
And for metabolism diseases, they are critical to differentiate. 

  In the DO, we follow the nomenclature developed by these research communities. 
  Rather than renaming disease terms. 
  Agreed it would be more convenient, but it would not be correct, nor would be be aligned with 
disease nomenclature. We work to align with proper nomenclature, thus we need to follow the community standards. 
In this example, their parentage makes it clear which term is inherited and which is acquired. 

As you mentioned in your comment, FlyBase and MGI have DO slims, that include these terms.
They curate their own slims, so I think it is safe to say that they know which term they are linking to.
They also pay close attending to parentage and definitions. 

Thank you for you suggested changes to the DO.
However, we are unable to make a etiology-neutral carbohydrate metabolism disease, as this would not represent an actual disease. 

Adding GO OWL def: thank you for the suggestion, we will take a look. And can plan for this for future work, currently it is not in our plans to add GO to the DO. Perhaps this could be part of our next grant. 

Cheers,
Lynn

**Comment by sbello** [comment]
Hi Chris,
The MGI_slim tag is an error. I must have missed that one when I cleaned up the extra tags that had been put in the DO. We have the parent terms 'inherited metabolic disorder' and 'acquired metabolic disease' in the MGI slim, so there is no need to have a tag on the child term. Next time I get in the file I'll clean that up. The MGI code for display ignores slim tags that are children of  other slim terms so this has no real impact on our display.
Thanks for the catch.
Sue

**Comment by sbello** [comment]
Hi Lynn,
I took a look at the terms Chris is referring to and I think we may want to modify the definitions to make it clear that these are not solely genetic in eitiology. I followed the definition template when entering these and I think they give an incorrect impression that the mutation is the sole cause of the gene. Could we discuss this on our next call?
Thanks,
Sue

**Comment by lschriml** [comment]
Hello Sue,
 Sounds good.

Cheers,
Lynn
